database/entity/0001-init_db/User.ts

```diff
@@ -3,22 +3,28 @@ import { BaseEntity, Entity, PrimaryGeneratedColumn, Column } from 'typeorm'
 // Moriz: I do not like the idea of having two user tables
 @Entity('state_users')
 export class User extends BaseEntity {
-  @PrimaryGeneratedColumn()
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
   id: number
 
+  @Column({ default: 0, name: 'index_id' })
```

Comment:
can we have an order defined here? I suggest to have `name` as first value if we set it
```suggestion
  @Column({ name: 'index_id', default: 0 })
```